ui/src/app/pages/workspace/research-purpose.tsx

```diff
@@ -57,7 +57,8 @@ const styles = reactStyles({
 export const ResearchPurpose = withCurrentWorkspace()(
   ({workspace}: {workspace: WorkspaceData}) => {
     const isOwner = WorkspacePermissionsUtil.isOwner(workspace.accessLevel);
-    const selectedResearchPurposeItems = getSelectedResearchPurposeItems(workspace.researchPurpose);
+    const selectedResearchPurposeItems = getSelectedResearchPurposeItems(workspace.researchPurpose, true);
+    const selectedPrimaryPurposeItems = getSelectedResearchPurposeItems(workspace.researchPurpose, false);
```

Comment:
open for name suggestion: selectedPrimaryPurpose seems more like all the primary purpose but the name should reflect  all Primary Purpose except Research Purpose